subprojects/packagefiles/libxslt/meson.build

```diff
@@ -97,6 +97,15 @@ endif
 
 libxml2_dep = dependency('libxml-2.0', version: '>=2.9.10')
 with_modules = libxml2_dep.get_variable(pkgconfig: 'modules', internal: 'modules', default_value: '0').to_int() != 0
+if host_machine.system() == 'cygwin'
+  config.set('MODULE_EXTENSION', '.cygdll')
```

Comment:
Upstream's Autotools has this as just `.dll`.

Reply:
Sounds like an upstream problem

Reply:
While I don't have convenient access to a Cygwin system, I can't find any evidence that Meson builds Cygwin shared libraries with a `.cygdll` extension.  Why do you think this shouldn't be `.dll`?

Reply:
I have one. No cygdll extension, just dll. Removed.

Reply:
Wonder if this is outdated: https://github.com/samtools/htslib/blob/master/configure.ac#L273

Reply:
I suppose `dlopen()` and equivalent functions don't actually care about the file extension of the specified file, so arbitrary extensions will work if a library decides to use them.